Examples/MAX78002/CameraIF/project.mk

```diff
@@ -15,16 +15,21 @@
 # connected camera.  These are some common values.  
 # For a full list of options for the 'CAMERA' variable, 
 # see the documentation.
-CAMERA=OV7692
-#CAMERA=OV5640
-# CAMERA=HM0360_MONO
+#CAMERA=OV7692
+CAMERA=OV5640
+#CAMERA=OV5642
+#CAMERA=HM0360_MONO
 #CAMERA=HM01B0
 
+ifeq "$(CAMERA)" "OV5640"
+PROJ_CFLAGS+=-DOV5640_DVP
```

Comment:
Maybe we can move this into the MAX78000 `board.mk` files, since the AI85 _only_ has a DVP interface

Reply:
Sure

Reply:
Done